identifiers/country-my.csv

```diff
@@ -72,7 +72,7 @@ ocd-division/country:my/state:05/ed:p128,Seremban
 ocd-division/country:my/state:05/ed:p129,Kuala Pilah
 ocd-division/country:my/state:05/ed:p130,Rasah
 ocd-division/country:my/state:05/ed:p131,Rembau
-ocd-division/country:my/state:05/ed:p132,Telok Kemang
+ocd-division/country:my/state:05/ed:p132,Port Dickson

```

Comment:
I'm not sure if we should continue to use the district numbering to identify these districts. From what I can tell these aren't renaming of the existing districts but the districts actually being replaced. Example:
- https://ms.wikipedia.org/wiki/Telok_Kemang_(kawasan_persekutuan)
- https://ms.wikipedia.org/wiki/Port_Dickson_(kawasan_persekutuan)

So I don't think it would be valid to say that an officeholder that represented Telok Kemang one term and then Port Dickson the next is representing the same district across both terms, and with the current identifiers we can't mark Telok Kemang with a `validTo` date.

What do you think about using the name as the electoral district identiifer?

Looping in @jpmckinney if he has any thoughts on this as well.

Reply:
For context, in the US, there are states (if I remember correctly!) with congressional districts 1, 2, 3, etc. They don't have names – only numbers – and a redistricting *could* change their boundary entirely – from one part of the state to another. To preserve the identity of CD 1 for a given period, we *could* suffix the number with the year of the redistricting. By not adding this suffix, we treat CD 1 as persisting over time. Since congressional districts (given the possibility of being redrawn on an entirely separate territory), have no real identity (whether at a given time or across time), this seems like an okay compromise.

In the case of Malaysia, as I understand, the districts have names and they can come into and go out of existence (unlike the plain numbered districts in the US). However, the data source (what is the data source?), I guess, assigns sequential numbers each time, with no concern for reassigning numbers. In that case, I think it's better to use the names for IDs.

Reply:
Thanks both. I have modified the files and use the district name as the id, and adding aliases.csv for backwards compatability. PTAL, thanks!